js/src/forum/addDiscussPrivatelyControl.js

```diff
@@ -10,7 +10,7 @@ export default function () {
         if (app.session.user &&
             app.session.user.id() !== user.id() &&
             app.forum.attribute('canStartPrivateDiscussion') &&
-            (user.blocksPd() === false || app.forum.attribute('canStartPrivateDiscussionWithBlockers'))
+            ((user.blocksPd() === 0 || app.forum.attribute('canStartPrivateDiscussionWithBlockers')) && !user.cannotBeDirectMessaged())
```

Comment:
Fixed this https://github.com/FriendsOfFlarum/byobu/commit/ea4a64a47ec84b37c2576a500fbb656a19866a8a

Please revert the false => 0 change